.github/workflows/discord_issues.yml

```diff
@@ -23,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "Discuss on Discord-Issues"
-        if: github.event.issue && contains(github.event.issue.labels.*.name, 'help wanted')
+        if: ${{ github.event.label.name == 'help wanted' }}
```

Comment:
Why can't we have exact match like here and have two steps (one for issue and one for PR) as before ?

Reply:
@vfdev-5 So basically I tried the same but it seems difficult to control the label type when we try to do it in a single file with both issue and PR, so I think this is more predictable and reliable as before the change